sdk/main/include/Executable.h

```diff
@@ -0,0 +1,126 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef EXECUTABLE_H
+#define EXECUTABLE_H
+
+#include "AccountId.h"
+
+#include <chrono>
+#include <vector>
+
+namespace Hedera
+{
+class AccountBalance;
+class AccountBalanceQuery;
+class AccountCreateTransaction;
+class Client;
+class TransactionResponse;
```

Comment:
`TransactionResponse` doesn't appear to be used anywhere. Is it for future use?

Reply:
Yup, this shouldn't be here. Removed